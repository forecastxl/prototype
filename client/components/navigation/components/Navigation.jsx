import React from 'react'
import styled from 'styled-components'
import { Link } from 'react-router'

const StyledLink = styled(Link)`
margin-left: 0.5rem;
margin-right: 0.5rem;
  &:first-of-type {
    margin-left: 0;
  }
  &:last-of-type {
    margin-right: 0;
  }
`

function Navigation() {
  return (
    <nav>
      <StyledLink to="/">Home</StyledLink>
      <StyledLink to="/login">Login</StyledLink>
      <StyledLink to="/signup">Signup</StyledLink>
    </nav>
  )
}

export default Navigation
