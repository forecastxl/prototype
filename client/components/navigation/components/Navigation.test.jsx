import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Navigation from './Navigation'

describe('<Navigation />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Navigation />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
