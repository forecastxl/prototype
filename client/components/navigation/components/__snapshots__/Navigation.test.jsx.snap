exports[`<Navigation /> renders correctly 1`] = `
<nav>
  <Styled(undefined)
    to="/">
    Home
  </Styled(undefined)>
  <Styled(undefined)
    to="/login">
    Login
  </Styled(undefined)>
  <Styled(undefined)
    to="/signup">
    Signup
  </Styled(undefined)>
</nav>
`;
